monolithic/monolithic-layered/src/main/java/com/epam/training/microservices/monolithic/web/DiseaseController.java

```diff
@@ -30,7 +30,7 @@ public ViewAllTemplateParams getViewAllTemplateParams() {
   @Override
   public ViewSingleTemplateParams getViewSingleTemplateParams() {
     return ViewSingleTemplateParams.builder()
-        .title("Edit disease")
+        .title("Edit Disease")
```

Comment:
I've discovered a cosmetic issue thanks to tests and fixed it immediately.

Reply:
Sure!